test/utils.jl

```diff
@@ -3,6 +3,9 @@ using OptimalTransport
 using LinearAlgebra
 using Random
 using Test
+using Distributions
+
+include("../src/utils.jl")
```

Comment:
I guess you added this since `discretemeasure` is not exported anymore? This should be done differently though (the location might change and/or certain parts of utils.jl might require additional packages or other types that are not available if you just include it): Remove it
```suggestion
```
and just qualify all occurrences of `discretemeasure`, i.e., use `OptimalTransport.discretemeasure` (one could also load it explicitly with `using OptimalTransport: discretemeasure` but I prefer `OptimalTransport.discretemeasure` since it clearly indicates that the function is not exported and this is how we do it for all other non-exported functions).

Reply:
Good to know, thanks.